Buildings/Controls/OBC/FDE/DOAS/DehumMode.mo

```diff
@@ -0,0 +1,150 @@
+within Buildings.Controls.OBC.FDE.DOAS;
+block DehumMode
+  "This block calculates when dehumidification mode is active."
+
+  parameter Real dehumSet(
```

Comment:
Rename to phiThr. Relative humidity threshold above which dehumidification mode is enabled. Max should be 1. Set default to 0.6.